index.js

```diff
@@ -267,4 +263,13 @@ Emailer.admin = {
     }
 };
 
+Emailer.addMiddleware = function(data, callback) {
+    data.app.head('/emailer-mandrill/reply', function(req, res) {
+        res.sendStatus(200);
+    });
+    data.app.post('/emailer-mandrill/reply', Emailer.receive);
+
+    callback();
+};
+
```

Comment:
I'm new to NodeBB and couldn't find any useful documentation on the hook system.

What I needed to add was a public route that was not tied to the user's status. Would we be able to achieve this with `static:app.load`?